code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -762,6 +766,27 @@
 			transfer_contents(R, dest_belly)
 			return
 
+		else if(prob(transferchance_secondary) && transferlocation_secondary) //After the first potential mess getting into, run the secondary one which might be even bigger of a mess.
+			var/obj/belly/dest_belly
+			for(var/obj/belly/B as anything in owner.vore_organs)
+				if(B.name == transferlocation_secondary)
+					dest_belly = B
+					break
+
+			if(!dest_belly)
+				to_chat(owner, "<span class='warning'>Something went wrong with your belly transfer settings. Your <b>[lowertext(name)]</b> has had it's transfer chance and transfer location cleared as a precaution.</span>")
+				transferchance = 0
+				transferlocation = null
```

Comment:
This is still setting the primary to nothing rather than the secondary. Probably copy-paste error.

Reply:
Oops. Thank you.